### PR TITLE
Limit paper-listbox to two items, open menu upwards.

### PR DESCRIPTION
**Pull request by jalmeroth**
Fix for: home-assistant/home-assistant#6285

### CODE REVIEW

File: src/more-infos/more-info-media_player.html [hunk 1]
```diff
@@ -96,7 +100,7 @@
       <div class='controls layout horizontal justified'
            hidden$='[[computeHideSelectSource(isOff, supportsSelectSource)]]'>
         <iron-icon class="source-input" icon="mdi:login-variant"></iron-icon>
-        <paper-dropdown-menu class="flex source-input" label-float label='Source'>
+        <paper-dropdown-menu class="flex source-input" label-float label='Source' vertical-align="bottom">
```
Comment (andrey-git) [inline src/more-infos/more-info-media_player.html]: You can use the `dynamic-align` property instead  which will auto-pick direction with more space.

File: src/more-infos/more-info-media_player.html [hunk 2]
```diff
@@ -43,6 +43,10 @@
         margin-left: 10px;
       }
 
+      paper-dropdown-menu.source-input paper-listbox {
+        height: 96px;
```
Comment (andrey-git) [inline src/more-infos/more-info-media_player.html]: Why limit to 96px if there is enough space to show more?

File: src/more-infos/more-info-media_player.html (hunk 1 above)
Comment (jalmeroth) [reply src/more-infos/more-info-media_player.html]: Thanks, I changed this.

File: src/more-infos/more-info-media_player.html (hunk 2 above)
Comment (jalmeroth) [reply src/more-infos/more-info-media_player.html]: Well, the issue is that on mobile, the dropdown menu can't escape the paper-dialog-scrollable (?) and the overflow is cut-off. Limiting the size helps to make the menu work again, but this is probably not the best way to fix this. Any ideas?

Comment (balloob) [reply src/more-infos/more-info-media_player.html]: Don't have any ideas but we can't ship 96px height. We should find another solution.

Comment (andrey-git) [reply src/more-infos/more-info-media_player.html]: I just tried it. The scrollbox doesn't limit the dropdown. It extends all the way to the top of the screen.

Comment (jalmeroth) [reply src/more-infos/more-info-media_player.html]: img_5350
Well it doesn't seem to work in Chrome/Safari/Home Assistant.app on iOS though.

**Comment by balloob** [comment]
This PR has gone stale. We should also not limit to 2 items. Closing this.

**Pull request closed without merging**